recipes/Functions.py

```diff
@@ -82,6 +84,14 @@ def render_form_v2(self):
             height=300,
         )
 
+    def get_price_roundoff(self, state: dict) -> float:
```

Comment:
used get_price_roundoff as get_raw_price get called in get_price_roundoff and it rounds off the value to max(1, price).